src/components/MultiSelect/content.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getContent } from './helpers';
+import { StyledText } from './styled';
+import Chips from './chips';
+
+const Content = props => {
+    const { variant, value, chipVariant, readOnly, disabled, onDelete } = props;
+    const onDeleteCallback = disabled || readOnly ? null : onDelete;
```

Comment:
this logic is here and in the chips twice, then we have it repeated three times, can we put it in only one place?

Reply:
Done